tests/functional/test_contract_container.py

```diff
@@ -66,3 +66,14 @@ def test_deploy_proxy(
     # Show we get the implementation contract type using the proxy address
     implementation = chain.contracts.instance_at(proxy.address)
     assert implementation.contract_type == vyper_contract_instance.contract_type
+
+
+def test_source_path(contract_container):
```

Comment:
we should split this into two tests, one for each condition in the `if` statement below